integration_tests/dbt_audit_helper/dbt_project.yml

```diff
@@ -0,0 +1,37 @@
+name: materialize_dbt_utils_dbt_utils_integration_tests
+version: '1.0'
+config-version: 2
+
+profile: integration_tests
+
+dispatch:
+  - macro_namespace: audit_helper
+    search_order: [materialize_dbt_utils, audit_helper]
+
+seeds:
+  dbt_utils_integration_tests:
+    schema_tests:
+      # Materialize does not support the `lead` window function.
+      data_test_mutually_exclusive_ranges_no_gaps:
+        +enabled: false
+      data_test_mutually_exclusive_ranges_with_gaps:
+        +enabled: false
+      data_test_mutually_exclusive_ranges_with_gaps_zero_length:
+        +enabled: false
+
+      # Materialize does not support the `lag` window function.
+      data_test_sequential_timestamps:
+        +enabled: false
+      data_test_sequential_values:
+        +enabled: false
+
+models:
+  dbt_utils_integration_tests:
+    datetime:
+      # Tested by test_date_spine_override.
+      test_date_spine:
+        +enabled: false
+    schema_tests:
+      # Tested by test_recency_override.
+      test_recency:
+        +enabled: false
```

Comment:
Not positive, but I think all this stuff doesn't apply to audit helper!

```suggestion
```

Reply:
Ah! Need to give it another pass to double check there aren't more copy+paste leftovers!